source/guides/dashboard/runs.md

```diff
@@ -121,3 +121,14 @@ Runs that have been canceled or are in an errored state can be archived from the
   `https://dashboard.cypress.io/projects/{project ID}/runs/{run number}`
 - Click **{% fa fa-history %} Restore from archive**
   {% imgTag /img/dashboard/restore-from-archive.png "restore-from-archive" %}
+
+## Test Case History
+
+{% imgTag /img/dashboard/runs/dashboard-runs-details-sidebar-test-case-history.png "Screenshot of the Test Case History panel" %}
+
+When looking further into a test runs information, there is a new Test Case History panel. It allows users to be able to:
```

Comment:
I feel like we could elaborate on this feature more:

* Grabbing a screenshot that shows the feature off better. Kevin posted a really good one a while back: https://cypressio.slack.com/archives/C01018MFWKT/p1594753438000300
* Describing how we track test cases (by spec and title)
* Calling out that you can view the diff of a single test case
* Noting that we're not currently collecting source maps, so the diff will only show transpiled code.
* Noting that we require commit information for this to work